### ISSUE TITLE
n.context.getSession().record fails since if getSession() returns undefined.

### ISSUE THREAD

**Issue by madhuhc** [aws-observability/aws-rum-web#74]
`n.context.getSession().record` fails since if `getSession()` returns undefined.

**Comment by madhuhc** [comment]
```
        return this.context.getSession().record;
```

can be changes to

```
        return this.context.getSession()?.record || false;
```

In the below files

https://github.com/aws-observability/aws-rum-web/blob/1d461c924c188a2fe78e21e5a3dced89e58b42a0/src/plugins/event-plugins/FetchPlugin.ts#L88

https://github.com/aws-observability/aws-rum-web/blob/1d461c924c188a2fe78e21e5a3dced89e58b42a0/src/plugins/event-plugins/XhrPlugin.ts#L142

**Comment by madhuhc** [comment]
<img width="907" alt="Screen Shot 2021-12-22 at 11 10 01 AM" src="https://user-images.githubusercontent.com/2299130/147300758-2b9bb472-5098-44a7-abb6-29b6730e5b92.png">

**Comment by madhuhc** [comment]
https://github.com/aws-observability/aws-rum-web/pull/75

**Comment by qhanam** [comment]
Thank you! Resolved by #75
